be/src/olap/in_list_predicate.cpp

```diff
@@ -106,8 +106,8 @@ IN_LIST_PRED_EVALUATE(NotInListPredicate, ==)
         *size = new_size; \
     } \
 
-IN_LIST_PRED_COLUMN_BLOCK_EVALUATE(InListPredicate, ==)
-IN_LIST_PRED_COLUMN_BLOCK_EVALUATE(NotInListPredicate, !=)
+IN_LIST_PRED_COLUMN_BLOCK_EVALUATE(InListPredicate, !=)
+IN_LIST_PRED_COLUMN_BLOCK_EVALUATE(NotInListPredicate, ==)
```

Comment:
why not in is ==?

Reply:
`==` means if it equal with map.end(), if in is true, it will meet an element, it will **!=** end()